### ISSUE TITLE
Doesn't seem to reload images when the src attribute changes

### ISSUE THREAD

**Issue by courthead** [topaxi/ember-img-lazy#7]
I'm using ember-img-lazy for avatars on my site. When you switch from one user profile to another and the img component isn't destroyed but its src changes, ember-img-lazy doesn't seem to load the new image.

Does the component need something like this?

```
didReceiveAttrs() {
  this._super(...arguments);

  if (this._src !== this.attrs.src) {
    this.loadImage();
  }
},
```

**Comment by rveitch** [comment]
This has been an issue for us as well.

**Comment by rveitch** [comment]
I've submitted a PR for this based on the suggestion from @courthead, but using this.getAttr('src') as referencing by `this.attrs.*` is against current Ember BP according to https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-attrs-in-components.md.

**Comment by topaxi** [comment]
Fix by @rveitch has been merged and published as `3.1.0`, let me know if any issues arise as I'm currently not very involved in the ember community or using ember in any project myself.
